CONTRIBUTING.md

```diff
@@ -18,43 +35,23 @@ You must agree to abide by the [Mozilla Community Participation Guidelines](http
-
-The FxA core team uses ZenHub to measure velocity and organize work into Epics.
-
-### Estimation and Point Values
+## Development Process
 
-Points are assigned to GitHub issues (ideally before starting work 😉) so that we can track our velocity over time, which aids in planning. Note that you'll need to be signed in to ZenHub to view and set estimates.
-
-We don't assign points to pull requests, unless it's a pull request without a corresponding issue.
-
-The goal of estimation is for us to assess the issue in terms of its relative complexity, effort, and doubt. When applying an estimate, we should consider all the steps in getting the particular work to a completed (ready for production) state for our consumer. The should include effort required for code reviews, security reviews, testing, integration and build/deploy.
-
-| Points  | Relative Size | Description                                                                |
-| ------- | ------------- | -------------------------------------------------------------------------- |
-| 1       | xs            | This is a trivial change with clearly defined parameters.                  |
-| 2       | s             | This is a small change with clearly defined parameters.                    |
-| 3       | m             | This is a small change, but there's some uncertainty.                      |
-| 5       | l             | Moderately complex, will require some effort but clearly defined.          |
-| 8       | xl            | Moderately complex, medium effort, some uncertainty.                       |
-| 13      | xxl           | Complex, large effort, well defined (these should be broken down further)  |
-| 21 (20) | ∞             | Meta issue or We do not have clear scope. (This issue must be broken down) |
+[Read about our development process](https://mozilla.github.io/ecosystem-platform/docs/fxa-engineering/fxa-dev-process)
 
 ## Sending Pull Requests
 
 Patches should be submitted as pull requests (PR).
 
 Before submitting a PR:
 
-- Your code must run and pass all the automated tests before you submit your PR for review. "Work in progress" pull requests are allowed to be submitted, but should be clearly labeled as such and should not be merged until all tests pass and the code has been reviewed.
+- Your code must run and pass all the automated tests before you submit your PR for review. "Work in progress" pull requests are allowed to be submitted, but should be marked `draft` and should not be merged until all tests pass and the code has been reviewed.
 - Your patch should include new tests that cover your changes. It is your and your reviewer's responsibility to ensure your patch includes adequate tests.
 - Your patch must be [GPG signed](https://help.github.com/articles/managing-commit-signature-verification) to ensure the commits come from a trusted source.
 
 When submitting a PR:
```

Comment:
One thing I think contributors sometimes don't do is flag @ fxa-devs, or some developer for review. This might happen on initial PR or when they fix followup issues. These PRs might sit around longer than usual.

Using the mark as draft/ ready to review would help reduce this.

Reply:
We're also going to start looking for PRs for anything over a couple days old in our weekly triage which should help catch any stragglers. w